FrontEnd/src/components/profileList/ProfileList.jsx

```diff
@@ -0,0 +1,59 @@
+import css from "./ProfileList.module.css";
+import { List, ConfigProvider } from "antd";
+import ProfileCard from "./ProfileCard";
+
+// FIXME: change loader to the one from dev
+// FIXME: change error page to the one from dev
+
+const ListHeader = ({ number }) => (
+  <div className={css["results-header"]}>
+    <p className={css["results-header__number"]}>{number}</p>
+    <p className={css["results-header__text"]}>компаній</p>
+  </div>
+);
+
+export default function ProfileList(props) {
+  const isAuthorized = props.isAuthorized;
+  const currentPage = props.current;
+  const data = props.data;
+  const companiesFound = data.total_items;
+  const profiles = data.results;
+  const pageSize = 6;
+
+  return (
+    <>
+      <ConfigProvider
+        theme={{
+          components: {
+            Pagination: {
+              colorPrimary: "#1F9A7C",
+              colorPrimaryHover: "#0b6c61",
+            },
+          },
+        }}
+      >
```

Comment:
No need to specify it every time it should be done once for the entire app

Reply:
Done